services/web-app/layouts/layout/layout.module.scss

```diff
@@ -11,6 +11,11 @@
   background: $background;
 }
 
+:global(.cds--header__nav::before) {
+  // override Carbon, hide divider line
+  display: none;
+}
```

Comment:
Small detail, but I think it's a good practice for any `:global()` to still be scoped to a moduled class name. For example, in `catalog-filters.module.scss`, we override `.cds--tag` but only inside the `.container` class, which gets locally scoped.

I know header nav is global and we only expect there to be one on the page, but to play it safe you could:

```scss
.header {
  :global(.cds--header__nav::before) {
    // override Carbon, hide divider line
    display: none;
  }
}
```

And `<Header aria-label="Carbon Design System" className={styles.header}>`.

To locally scope that Carbon override.

Reply:
Updated!